### PR TITLE
Add a simple test for sign up service

### PR DESCRIPTION
**Pull request by prashantkndl**

### CODE REVIEW

File: tests/authService.test.js [hunk 1]
```diff
@@ -14,4 +14,16 @@ test("generateSecret", (done) => {
...
+
+test("Test Signup service", (done) => {
+    const userModel = require("./mocks/mockUserModel");
+    const auth = authService.authService(userModel);
+
+    auth.signUp("sample@example.com", "Sample Example",  (ERRCODE, result) => {
+        expect(ERRCODE).toBeNull();
+        expect(result).not.toBeNull();
+        expect(result.secret).not.toBeNull();
+        done()
```
Comment (bishalspkt) [inline tests/authService.test.js]:
```suggestion
        done();
```

File: tests/mocks/mockUserModel.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+"use strict";
+
+/**
+ * Barebones mock user model
+ */
+const userModel = () => ({ save: (cb) => cb() })
```
Comment (bishalspkt) [inline tests/mocks/mockUserModel.js]:
```suggestion
const userModel = () => ({ save: (cb) => cb() });
```

File: tests/mocks/mockUserModel.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+userModel.findOne = function(searchObject, cb) {
+    cb(null, null);
+}
```
Comment (bishalspkt) [inline tests/mocks/mockUserModel.js]:
```suggestion
};
```

**Comment by prashantkndl** [comment]
I will push a commit with these changes

**Pull request merged**